src/resolve/resolved-varlink.c

```diff
@@ -519,6 +535,31 @@ static int vl_method_resolve_address(Varlink *link, JsonVariant *parameters, Var
         return 1;
 }
 
+static int vl_method_subscribe_dns_resolves(Varlink *link, JsonVariant *parameters, VarlinkMethodFlags flags, void *userdata)
+{
+        Manager *m;
+        int r;
+
+        assert(link);
+
+        m = varlink_server_get_userdata(varlink_get_server(link));
+        assert(m);
```

Comment:
Why not to directly use the `userdata` argument of this function?

Reply:
I actually just followed the convention in other similar varlink handlers, where `userdata` isn't used. I believe we are only setting the userdata for the `VarlinkServer` object, not individual varlink methods. So in this case, the `userdata` actually is `NULL`

Reply:
Indeed. My bad. Sorry for the noise.